letta/constants.py

```diff
@@ -48,6 +48,9 @@
 DEFAULT_MESSAGE_TOOL = "send_message"
 DEFAULT_MESSAGE_TOOL_KWARG = "message"
 
+# Structured output models
+STRUCTURED_OUTPUT_MODELS = {"gpt-4o-mini-2024-07-18", "gpt-4o-2024-08-06"}
```

Comment:
would this fail to match if the model name is just `gpt-4o-mini`?

Reply:
changed it to be less strict (4o or 4o-mini)